templates/circle-config.js

```diff
@@ -0,0 +1,39 @@
+module.exports = () => {
+return `version: 2
```

Comment:
We have a new circle config now 🙈

Reply:
https://raw.githubusercontent.com/Financial-Times/o-table/master/.circleci/config.yml

Reply:
_done_